docs/operations-repo-structure.md

```diff
@@ -41,19 +41,19 @@ BitOps expects an operations repo to be in the following structure, where each e
         └── bitops.config.yaml
 ```
 #### Environment Directories
-These directories live at the root of an operations repository and are used to separate applications and environments. Depending on your usecase, you may have an environment for `production`, `test` and `dev` or these traditional environments may be further separated into individual services such as `test_service_a` and `test_serice_b`. This pattern is preferential to having a branch for each environment as this allows the state of all your infrastructure to be managed from one location without merging potentially breaking an environment.
+These directories live at the root of an operations repository and are used to separate applications and environments. Depending on your use case, you may have an environment for `production`, `test` and `dev` or these traditional environments may be further separated into individual services such as `test_service_a` and `test_serice_b`. This pattern is preferential to have a branch for each environment as this allows the state of all your infrastructure to be managed from one location without merging potentially breaking an environment.
```

Comment:
this was correct
> This pattern is preferential to having a branch

Reply:
Thanks for going through all the changes 👍 
Reverted in https://github.com/bitovi/bitops/pull/281/commits/925eb2443597b49d00aec9bfbd8c25f9d66cfbc9